speech_recognition_msgs/msg/WordInfo.msg

```diff
@@ -0,0 +1,7 @@
+# please see below URL for more details.
+# https://cloud.google.com/speech-to-text/docs/reference/rest/v1/speech/recognize#wordinfo
+float64 start_time
+float64 end_time
```

Comment:
`start_time` is duration from the initial time of sentences to the start of the word. This is not clear from the field name itself.

Please add field description or change field name in order to make target use of each field clear to users.